File: notes/templates/desktop/qt5_qt6/qt6/gui.cpp
#include "gui.h"
#include "ui_gui.h"

Gui::Gui(QWidget *parent)
    : QMainWindow(parent)
    , ui(new Ui::Gui)
{
    ui->setupUi(this);
    setFixedSize(200,100);

    connect(ui->btnQuit,SIGNAL(clicked()),this,SLOT(btnQuit_clicked()));
    connect(ui->btnMsg,SIGNAL(clicked()),this,SLOT(btnMsg_clicked()));
}

Gui::~Gui()
{
    delete ui;
}


void Gui::btnQuit_clicked()
{
    QApplication::quit();
}


void Gui::btnMsg_clicked()
{
    QMessageBox::information(this,"Template","Template C++ Qt",QMessageBox::Ok);
}

File: notes/templates/desktop/qt/qt6-pyside/gui.py
# -*- coding: utf-8 -*-

import sys
from PySide6.QtWidgets import QMainWindow,QMessageBox
from ui_gui import Ui_Gui

class Gui(QMainWindow,Ui_Gui):
    def __init__(self, parent=None):
        super(Gui, self).__init__(parent)
        self.setupUi(self)
        self.setFixedSize(200,100)

        self.btnMsg.clicked.connect(self.btnMsg_clicked)
        self.btnQuit.clicked.connect(self.btnQuit_clicked)

    def btnMsg_clicked(self):
        msgbox = QMessageBox()
        msgbox.setIcon(QMessageBox.Information)
        msgbox.setWindowTitle("Template")
        msgbox.setText("Template Qt PySide")
        msgbox.setStandardButtons(QMessageBox.Ok)
        msgbox.exec()

    def btnQuit_clicked(self):
        sys.exit()

